src/renderer/viz/expressions/viewportFeatures.js

```diff
@@ -54,6 +56,7 @@ import CartoRuntimeError from '../../../errors/carto-runtime-error';
 export default class ViewportFeatures extends BaseExpression {
     constructor (...properties) {
         properties = properties.map(p => implicitCast(p));
+        // TODO validate here properties don't collide with interactivity.Feature predefined props
```

Comment:
Having properties and viz properties under the same namespace may bite us in the future. But changing this would be a breaking change to add another namespace... I don't know what should we do.

cc: @rochoa

Reply:
I thought about a breaking change here. 

IMO it's more clear if we move the vizProperties one level down (as the current `variables`), and keep the "feature properties" there. For now I added a validation to catch this and raise an error (e.g. when trying to use a 'color' column from a feature)